tests/sota_tools/test-garage-deploy-dry-run

```diff
@@ -12,9 +12,9 @@ TREEHUB="{\
 }"
 
 echo $TREEHUB > $TEMP_DIR/treehub.json
-./tests/sota_tools/treehub_server.py $PORT &
-sleep 1
+./tests/sota_tools/treehub_server.py -p${PORT} -c &
```

Comment:
Do you not need a space after the `-p`? If not, no problem.

Reply:
no, should work either way